### DISCUSSION TITLE
Does the ARC runner have ETCD on it?

### DISCUSSION THREAD

**Discussion by sd185434** [actions/actions-runner-controller#3385] [FAQs]
Hello everyone, hope everyone is ok. My team is working on creating a etcd to have a key value pair that can be accessed at a run level and we were wondering if the ARC runner or if the runner pods have an etcd instance that we can use for this or if it would be better if we create one ourselves? Thanks so much

**Comment by nikola-jokic** [comment]
The runner doesn't come with the etcd instance with it. I'm not sure what image you are using, but both the [official runner image](https://github.com/actions/runner/blob/main/images/Dockerfile) and [community ones](https://github.com/actions/actions-runner-controller/tree/master/runner) are not pre-installing the etcd.
